### PR TITLE
Introduce sensible weight constants

### PR DESCRIPTION
**Pull request by KiChjang**
This PR removes the `WEIGHT_PER_{SECOND, MILLIS, MICROS, NANOS}` (originally of type `Weight`) and replaces them with `WEIGHT_REF_TIME_PER_*` `u64`s and introduces `WEIGHT_PROOF_SIZE_PER_{MB, KB}`, also `u64`s.

polkadot companion: paritytech/polkadot#6406
cumulus companion: paritytech/cumulus#1970

### CODE REVIEW

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-check-benches
**Logs**: https://gitlab.parity.io/parity/mirrors/substrate/-/jobs/2129601

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/substrate/-/jobs/2129751

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/substrate/-/jobs/2129752

**Review by sam0x17 — APPROVED** [review]

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/substrate/-/jobs/2130258

**Review by gpestana — APPROVED** [review]

File: primitives/weights/src/lib.rs [hunk 1]
```diff
@@ -47,12 +47,13 @@ pub use weight_meter::*;
...
+	pub const WEIGHT_REF_TIME_PER_MILLIS: u64 = 1_000_000_000;
+	pub const WEIGHT_REF_TIME_PER_MICROS: u64 = 1_000_000;
+	pub const WEIGHT_REF_TIME_PER_NANOS: u64 = 1_000;
 
-	pub const WEIGHT_PER_SECOND: Weight = Weight::from_ref_time(1_000_000_000_000);
-	pub const WEIGHT_PER_MILLIS: Weight = Weight::from_ref_time(1_000_000_000);
-	pub const WEIGHT_PER_MICROS: Weight = Weight::from_ref_time(1_000_000);
-	pub const WEIGHT_PER_NANOS: Weight = Weight::from_ref_time(1_000);
+	pub const WEIGHT_PROOF_SIZE_PER_MB: u64 = 1024 * 1024;
+	pub const WEIGHT_PROOF_SIZE_PER_KB: u64 = 1024;
```
Comment (ggwpez) [inline primitives/weights/src/lib.rs:56]: I considered this as well, thanks!

**Comment by ggwpez** [comment]
/cmd queue -c bench-bot $ pallet dev pallet_lottery

**Comment by KiChjang** [comment]
bot merge

**Comment by Polkadot-Forum** [comment]
This pull request has been mentioned on **Polkadot Forum**. There might be relevant details there:

https://forum.polkadot.network/t/polkadot-release-analysis-v0-9-36/1529/1

**Pull request merged**